tests/xua_sim_tests/test_mixer_routing_input.py

```diff
@@ -22,13 +21,14 @@ def do_test(options, capfd, test_file, test_seed):
         str(max_cycles),
     ]
 
-    build_options = []
-    build_options += ["TEST_SEED=" + str(test_seed)]
+    seed_hdr = Path(__file__).parent / testname / "src" / "test_seed.h"
+    with open(seed_hdr, "w") as f:
+        f.write(f"#define TEST_SEED ({test_seed})")
 
     result = Pyxsim.run_on_simulator(
         binary,
+        cmake=True,
```

Comment:
Ah is this why you had a fork? Nice feature if so - need to get that merged and let people know about it